File: commonLib/src/main/java/com/travel/commonlib/hotel/SearchIdResponse.java
package com.travel.commonlib.hotel;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class SearchIdResponse {
    @SerializedName("SearchId")
    @Expose
    public String searchId;

    public String getSearchId() {
        return searchId;
    }

    public void setSearchId(String searchId) {
        this.searchId = searchId;
    }
}

File: commonLib/src/main/java/com/travel/commonlib/activity/searchkey/SearchIdResponse.kt
package com.travel.commonlib.activity.searchkey

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

class SearchIdResponse : Serializable {
    @SerializedName("SearchId")
    @Expose
    var searchId: String? = null

    companion object {
        private const val serialVersionUID = 6866886448338009716L
    }
}